modules/collectionRegistrationMixin.js

```diff
@@ -213,7 +213,7 @@
        * Removes a requester from this cache. No longer receives updates
```

Comment:
update line 212  and 160 to reflect this change.

Also run gulp bundle to update the bundle